state-indexer/src/configs.rs

```diff
@@ -543,6 +551,7 @@ impl ScyllaDBManager {
         Ok(result)
     }
 
+    #[cfg(feature = "account_access_keys")]
```

Comment:
I've disabled this function completely if the feature is disabled (by default). Also, I've used the full path to `HashMap` to avoid redundant feature-flag with the import at the top.